src/components/cards/BaseStakingCard.tsx

```diff
@@ -517,12 +517,27 @@ const BaseStakingCard: FC<IBaseStakingCardProps> = ({
 											setShowUnStakeModal(true)
 										}
 									/>
-									<StakeAmount>
+									<StakeAmount gap='5px'>
 										{type === StakingType.UNISWAPV3_ETH_GIV
 											? `${stakedLpAmount.toNumber()} ${unit}`
 											: `${formatWeiHelper(
 													stakedLpAmount,
 											  )} ${unit}`}
+										{isLocked && (
+											<IconWithTooltip
+												icon={<IconHelp size={16} />}
+												direction={'right'}
```

Comment:
add right direction in that position, may cause adding horizontal scroll to the page like this:
![Screenshot from 2022-08-03 14-05-06](https://user-images.githubusercontent.com/17097576/182576183-7022124f-ca0d-48c2-a741-2ba4ac4b37bb.png)
I suggest change it to left

Reply:
ahh! good to know, changed it to top so it doesn't block the text the tooltip is telling you to click :)